src/NuGet.Clients/NuGet.Console/NuGet.Console.csproj

```diff
@@ -1,157 +1,30 @@
-    <Compile Include="IConsoleInitializer.cs" />
-    <Compile Include="IHostMetadata.cs" />
-    <Compile Include="IHostProvider.cs" />
-    <Compile Include="IPathExpansion.cs" />
-    <Compile Include="IPowerConsoleWindow.cs" />
-    <Compile Include="IPSNuGetProjectContext.cs" />
-    <Compile Include="ITabExpansion.cs" />
-    <Compile Include="IWpfConsole.cs" />
-    <Compile Include="IWpfConsoleService.cs" />
-    <Compile Include="OutputConsole\ChannelOutputConsole.cs" />
-    <Compile Include="OutputConsole\BuildOutputConsole.cs" />
-    <Compile Include="OutputConsole\OutputConsole.cs" />
-    <Compile Include="OutputConsole\OutputConsoleProvider.cs" />
-    <Compile Include="OutputConsole\BaseOutputConsole.cs" />
-    <Compile Include="OutputConsole\SharedOutputConsole.cs" />
-    <Compile Include="ScriptExecutor.cs" />
-    <Compile Include="SimpleExpansion.cs" />
-    <Compile Include="Token.cs" />
-    <Compile Include="TokenType.cs" />
-    <Compile Include="Utils\CommandUiUtilities.cs" />
-    <Compile Include="Xamls\ConsoleContainer.xaml.cs">
-      <DependentUpon>ConsoleContainer.xaml</DependentUpon>
-    </Compile>
-    <Compile Include="Console\IPrivateConsoleStatus.cs" />
-    <Compile Include="Guids.cs" />
-    <Compile Include="IHostInitializer.cs" />
-    <Compile Include="IPowerConsoleService.cs" />
-    <Compile Include="NativeMethods.cs" />
-    <Compile Include="PkgCmdID.cs" />
-    <Compile Include="Utils\CommonExtensionMethods.cs" />
-    <Compile Include="Utils\ExtensionMethods.cs" />
-    <Compile Include="Utils\Marshaler.cs" />
-    <Compile Include="Utils\ObjectWithFactory.cs" />
-    <Compile Include="Utils\UtilityMethods.cs" />
-    <Compile Include="Console\ConsoleDispatcher.cs" />
-    <Compile Include="Console\IHostNameMetadata.cs" />
-    <Compile Include="Console\InputHistory.cs" />
-    <Compile Include="Console\OleCommandFilter.cs" />
-    <Compile Include="Console\OrderedSpans.cs" />
-    <Compile Include="Console\TextFormatClassifier.cs" />
-    <Compile Include="WpfConsole\WpfConsole.cs" />
-    <Compile Include="WpfConsole\WpfConsoleClassifier.cs" />
-    <Compile Include="WpfConsole\WpfConsoleService.cs" />
-    <Compile Include="WpfConsole\WpfConsoleCompletionSource.cs" />
-    <Compile Include="WpfConsole\WpfConsoleKeyProcessor.cs" />
-    <Compile Include="PowerConsoleToolWindow.cs" />
-    <Compile Include="PowerConsole\ClassifierProvider.cs" />
-    <Compile Include="PowerConsole\CompletionSourceProvider.cs" />
-    <Compile Include="PowerConsole\HostInfo.cs" />
-    <Compile Include="PowerConsole\PowerConsoleWindow.cs" />
-    <Compile Include="Resources.Designer.cs">
-      <AutoGen>True</AutoGen>
-      <DesignTime>True</DesignTime>
-      <DependentUpon>Resources.resx</DependentUpon>
-    </Compile>
-    <Compile Include="GlobalSuppressions.cs" />
-    <Compile Include="Properties\AssemblyInfo.cs" />
-  </ItemGroup>
-  <ItemGroup>
-    <EmbeddedResource Include="Resources.resx">
```

Comment:
SDK-style projects usually need an `<EmeddedResource Update="" />` to make things look right in VS.  

https://github.com/NuGet/NuGet.Client/blob/dev/src/NuGet.Core/NuGet.Versioning/NuGet.Versioning.csproj#L20-L33